dhcpv6/option_bootfileparam.go

```diff
@@ -0,0 +1,68 @@
+package dhcpv6
+
+import (
+	"fmt"
+
+	"github.com/u-root/u-root/pkg/uio"
+)
+
+// OptBootFileParam implements the OptionBootfileParam option
+//
+// This module defines the OPT_BOOTFILE_PARAM structure.
+// https://www.ietf.org/rfc/rfc5970.txt (section 3.2)
+type OptBootFileParam []string
+
+var _ Option = OptBootFileParam(nil)
+
+// Code returns the option code
+func (op OptBootFileParam) Code() OptionCode {
+	return OptionBootfileParam
+}
+
+// ToBytes serializes the option and returns it as a sequence of bytes
+func (op OptBootFileParam) ToBytes() []byte {
+	buf := uio.NewBigEndianBuffer(nil)
+	for _, param := range op {
+		if len(param) >= 1<<16 {
+			panic("a too long parameter")
```

Comment:
I don't think panicking is OK here. this would be the only `panic()` outside of a test in this entire library. (this is the same issue as the previous comment, that the ToBytes() api really should return an error). 
This might end up being acceptable, but we need to make a conscious decision that ToBytes() can panic and clearly communicate that to users

Reply:
Good catch. I totally missed this one.

Reply:
@Natolumin I agree that a `panic()` is bad. But what do you suggest? Just ignore an error? Because there's no `error` in returnings of `ToBytes()`.

Reply:
As per `hugelgupf` in https://github.com/insomniacslk/dhcp/pull/329:

```
I don't think this library should concern itself with validation of the values. My take is - the user is responsible.
```

I would keep it consistent with other options and don't panic in here.

Reply:
In my very-very humble opinion, it sounds terrible. Even standard Golang packages panics on unhandled errors. But OK. I will just drop huge parameters (use `continue` instead of `panic()`).